setup/scaling/caching/externalize-redis.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: single
+title:  "Externalize Redis"
+sidebar:
+  nav: setup
+---
+
+{% include toc %}
+
+One of easiest ways to improve Spinnaker's reliability at scale is to use an
+external Redis. The Redis installed by Spinnaker (either locally, or in
+Kubernetes) isn't configured to be production-ready. If you have a hosted Redis
```

Comment:
Hrm, we state in the setup guide that if you're not following one of the quickstarts, you're going to be installing Spinnaker potentially for use in production. Later in this guide, we say that the way it was actually installed isn't really production-ready, if I'm understanding this correctly.

Should the installation steps be updated to include this advice?

/cc @dorbin @nhayes

Reply:
Good point. @lwander ?

Reply:
https://github.com/spinnaker/spinnaker.github.io/pull/835

Reply:
Submitted suggestion

Reply:
merged. does this clear this PR to proceed?

Reply:
This PR was already merged before Andrew's feedback